github/resource_github_repository.go

```diff
@@ -87,7 +87,7 @@ func resourceGithubRepository() *schema.Resource {
 			"auto_init": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				ForceNew: true,
+				ForceNew: false,
```

Comment:
hi @majormoses! you can remove this `ForceNew` attribute all together since my default it's not set in a resource

Reply:
Makes sense 👍